Sources/GRPCHTTP2Core/GRPCStreamStateMachine.swift

```diff
@@ -398,14 +398,17 @@ struct GRPCStreamStateMachine {
   }
 
   /// The result of requesting the next outbound message.
-  enum OnNextOutboundMessage: Equatable {
+  enum OnNextOutboundMessage {
     /// Either the receiving party is closed, so we shouldn't send any more messages; or the sender is done
     /// writing messages (i.e. we are now closed).
     case noMoreMessages
     /// There isn't a message ready to be sent, but we could still receive more, so keep trying.
     case awaitMoreMessages
     /// A message is ready to be sent.
-    case sendMessage(ByteBuffer)
+    case sendMessage(
+      message: ByteBuffer,
```

Comment:
at this point it's no longer a message, it could be multiple framed messages (though I realise now that all of the naming here is message based...)

Reply:
Hm, you're right. I've renamed these to be more consistent although I did keep `awaitMoreMessages` and `noMoreMessages`, since I believe it's not wrong to refer to messages in those situations (as we're either waiting for more messages to arrive, regardless of how we frame them, or we know that no more messages will arrive).